### PR TITLE
Petya nikleva task 1/get started header footer list of items

### PR DESCRIPTION
**Pull request by petyaNikleva**
get started - created  header, footer and list of cities

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+body {
+  background-image: url('./../public/images/sky.jpg');
+  background-color: #cccccc;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100vh;
+}
```
Comment (virzhinia-petkova-redshelf) [inline src/App.css]: do you need two classes with the same name? Should you just keep all styles for body in one place?

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import DropDownMenu from './components/DropDownMenu/DropDownMenu';
+import RandomText from './components/RandomText/RandomText';
+
+function App() {
+  return (
+    <div className="app-container">
+      <Header />
+      <main>
+        <DropDownMenu />
+        {/* <RandomText />   */}
```
Comment (virzhinia-petkova-redshelf) [inline src/App.js]: can you remove this if it's not needed

File: src/components/DropDownMenu/DropDownMenu.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import styles from './DropDownMenu.module.css';
+
+import initialCities from '../../storage/initialCities';
+
+function DropDownMenu() {
+  const [searchValue, setSearchValue] = useState(null);
+
+  function getSearchedValue(e) {
+    setSearchValue(e.target.value);
+  }
```
Comment (virzhinia-petkova-redshelf) [inline src/components/DropDownMenu/DropDownMenu.js]: it's good to keep a consistent style throughout your application. So I'd recommend converting this function(and other function declarations) to an arrow function.

File: src/components/DropDownMenu/DropDownMenu.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+  const citiesToShow = searchValue
+    ? initialCities.filter(city => city.cityName.includes(searchValue))
+    : initialCities;
+
+  return (
+    <div className={styles.dropdownMenuContainer}>
+      <label>
+        Choose a city from this list:
```
Comment (virzhinia-petkova-redshelf) [inline src/components/DropDownMenu/DropDownMenu.js]: can this live in a file ./constants in the DropDownMenu folder?

File: src/components/Footer/Footer.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  function getTime() {
+    const currentdate = new Date();
+    const hour = getLocalHours(currentdate);
+    const min = getLocalMinutes(currentdate);
+    const sec = getLocalSeconds(currentdate);
+    const currentTime = hour + ':' + min + ':' + sec;
+    return currentTime;
+  }
+
+  function getLocalHours(currentdate) {
```
Comment (virzhinia-petkova-redshelf) [inline src/components/Footer/Footer.js]: having your function after it's already been used is not a great practice. It's difficult to read. Also does it make sense to move functions like this one to a file called helpers and export them from there?

File: src/components/Footer/Footer.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    const sec = currentdate.getSeconds();
+    if (sec < 10) {
+      return `0${sec}`;
+    }
+    return sec;
+  }
+
+  return (
+    <footer className={styles.footer}>
+      <p>Current time: {time}</p>
```
Comment (virzhinia-petkova-redshelf) [inline src/components/Footer/Footer.js]: constant please :)

File: src/components/Header/Header.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+function Header() {
+  return (
+    <header className={styles.headerContainer}>
+      <img
+        className={styles.img}
+        src={require('../../../public/images/logo.png')}
+        alt="Logo image"
+      />
+      <h1> Show Me The Weather </h1>
```
Comment (virzhinia-petkova-redshelf) [inline src/components/Header/Header.js]: constant please

File: src/components/RandomText/RandomText.js [hunk 8]
```diff
@@ -0,0 +1,308 @@
+function RandomText() {
```
Comment (virzhinia-petkova-redshelf) [inline src/components/RandomText/RandomText.js]: do you need this entire file?

File: src/components/Footer/Footer.js [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+    return min;
+  }
+
+  function getLocalSeconds(currentdate) {
+    const sec = currentdate.getSeconds();
+    if (sec < 10) {
+      return `0${sec}`;
+    }
+    return sec;
+  }
```
Comment (virzhinia-petkova-redshelf) [inline src/components/Footer/Footer.js]: those two look pretty similar, can they be adjusted to be one function?

File: src/components/Footer/Footer.module.css [hunk 10]
```diff
@@ -0,0 +1,14 @@
+.footer {
+  background-color: #7209b7;
```
Comment (virzhinia-petkova-redshelf) [inline src/components/Footer/Footer.module.css]: do you think it's be a good idea to add the colors to variables?

Comment (virzhinia-petkova-redshelf) [reply src/components/Footer/Footer.module.css]: also you can group your css properties as it's a little difficult to follow the style - you can choose your own approach or see this suggestion - https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

**Review by virzhinia-petkova-redshelf — COMMENTED** [review]
Good job! A few comments from my side :)

File: src/App.css [hunk 11]
```diff
@@ -0,0 +1,29 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;0,900;1,500;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400;500;600;700&family=Lobster&display=swap');
+@import './reset.css';
+
+:root {
+  --pink: #b5179e;
+  --purple: #601293;
+  --blue: #3f37c9;
```
Comment (virzhinia-petkova-redshelf) [inline src/App.css]: some people use variable names like those but you can also take a look at this article https://medium.com/@stefanmorris/proper-naming-of-color-variables-683264fec0f4 . Up to you though.

File: src/components/DropDownMenu/DropDownMenu.js [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+  const getSearchedValue = e => setSearchValue(e.target.value);
+
+  const citiesToShow = searchValue
+    ? initialCities.filter(city => city.cityName.includes(searchValue))
+    : initialCities;
+
+  return (
+    <div className={styles.dropdownMenuContainer}>
+      <label>
+        Choose a city from this list:
```
Comment (virzhinia-petkova-redshelf) [inline src/components/DropDownMenu/DropDownMenu.js]: should this go into a contant?

File: src/components/helper/helper.js [hunk 13]
```diff
@@ -0,0 +1,18 @@
+const getLocalHours = currentdate => currentdate.getHours();
+
+const getLocalUnit = (currentdate, getUnit) => {
+  const unit = currentdate[getUnit]();
```
Comment (virzhinia-petkova-redshelf) [inline src/components/helper/helper.js:4]: nice :)

File: src/components/Footer/Footer.js (hunk 6 above)
Comment (virzhinia-petkova-redshelf) [reply src/components/Footer/Footer.js]: thoughts here?

File: src/components/DropDownMenu/DropDownMenu.js (hunk 12 above)
Comment (petyaNikleva) [reply src/components/DropDownMenu/DropDownMenu.js]: changes are addressed @virzhinia-petkova-redshelf

File: src/App.css (hunk 11 above)
Comment (petyaNikleva) [reply src/App.css]: Thanks for the link. I will try this approach. @virzhinia-petkova-redshelf

File: src/components/helper/helper.js (hunk 13 above)
Comment (petyaNikleva) [reply src/components/helper/helper.js:4]: Thanks!

File: src/components/DropDownMenu/DropDownMenu.js (hunk 12 above)
Comment (virzhinia-petkova-redshelf) [reply src/components/DropDownMenu/DropDownMenu.js]: did you decide against moving this to a constant or you forgot about it?

File: src/components/Footer/Footer.js (hunk 6 above)
Comment (virzhinia-petkova-redshelf) [reply src/components/Footer/Footer.js]: same question

File: src/components/DropDownMenu/DropDownMenu.js (hunk 12 above)
Comment (petyaNikleva) [reply src/components/DropDownMenu/DropDownMenu.js]: Sorry, I didn't get your idea initially. Now all is clear. The labels have been moved in a separate file as discussed. @virzhinia-petkova-redshelf

File: src/components/Footer/Footer.js (hunk 6 above)
Comment (petyaNikleva) [reply src/components/Footer/Footer.js]: The labels have been moved in a separate file as discussed. @virzhinia-petkova-redshelf

**Review by virzhinia-petkova-redshelf — APPROVED** [review]

**Pull request merged**